examples/chart/teleport-agent/templates/config.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Release.Name }}
+data:
+  teleport.yaml: |
+    teleport:
+      auth_token: "/etc/teleport-secrets/auth-token"
+      auth_servers: ["{{ required "proxyAddr is required in chart values" .Values.proxyAddr }}"]
+
+    kubernetes_service:
+      enabled: {{ ternary "true" "false" (contains "kube" (.Values.roles | toString))}}
+      {{- if contains "kube" (.Values.roles | toString) }}
```

Comment:
nit: I'd use an `if/else` pattern to avoid doing the roles check twice:
```
{{- if contains "kube" (.Values.roles | toString) }}
  enabled: true
  kube_cluster_name: ...
{{- else }}
  enabled: false
{{- end }}
```